### ISSUE TITLE
no ITS regions found?

### ISSUE THREAD

**Issue by gabrielet** [USDA-ARS-GBRU/q2_itsxpress#11]
so, I am running some test with the plugin and everything seems to be working fine except for the fact that I have en empty artifact both with my data and with the test data that I downloaded from this repo. This is what I am doing, nothing fancy:

```
qiime itsxpress trim-pair --i-per-sample-sequences ~/Downloads/paired.qza --p-region ALL --p-taxa F --o-trimmed ~/Downloads/its.qza
Saved SampleData[JoinedSequencesWithQuality] to: ~/Downloads/its.qza

qiime itsxpress trim-pair-output-unmerged --i-per-sample-sequences ~/Downloads/paired.qza --p-region ALL --p-taxa F --o-trimmed ~/Downloads/its_unmerged.qza
Saved SampleData[PairedEndSequencesWithQuality] to: ~/Downloads/its_unmerged.qza

```
of course I have two artifacts but the fastq files inside of them are empty. No ITS region was found there. What is happening? I have qiime2 2021.8 and bbmap 38.69, installed as mentioned [here](https://forum.qiime2.org/t/itsxpress-bbmerge-error/15807), after having the same error mentioned in the thread. I believe there must be issues with libraries, since no error was reported. For the same reason, i.e. no error, I can't figure out what went wrong. Any ideas?

These are more info:
```
$ pip show itsxpress
Name: itsxpress
Version: 1.8.0
Summary: Rapidly trim sequences down to their Internally Transcribed
Spacer (ITS) regions
Home-page: http://github.com/usda-ars-gbru/itsxpress
Author: Adam R. Rivers
Author-email: adam.rivers@ars.usda.gov
License: License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
Location:
~/miniconda3/envs/qiime2-2021.8/lib/python3.8/site-packages
Requires: biopython
Required-by: q2-itsxpress

$ pip show q2-itsxpress
Name: q2-itsxpress
Version: 1.8.0
Summary: A QIIME2 plugin to trim ITS regions using ITSxpress
Home-page: https://github.com/usda-ars-gbru/q2_itsxpress
Author: Adam R. Rivers, Kyle C. Weber
Author-email: adam.rivers@ars.usda.gov, kweber1@ufl.edu
License: License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
Location:
~/miniconda3/envs/qiime2-2021.8/lib/python3.8/site-packages
Requires: qiime2, pandas, itsxpress
Required-by:
```

**Comment by arivers** [comment]
What ITS region did you sequence and what primer set did you use?. It's
uncommon for users to use the --region all flag since most people target
its1 or its2. That may be causing the problem.

**Comment by gabrielet** [comment]
I totally missed that flag, and yes that is the problem, thank you for pointing it out!

So, I tried with the test set and it works, using the ITS2 region. The primers we used are gITS7 and ITS4. To make sure, I just tried to run both version of the function, with ITS1 and ITS2.

ITS2 provided me with some output, which means I found the sequences I was looking for, right?

Sorry for such a mistake! I thought one can extract both regions, but apparently that is not the case (i am still learning about fungal amplicons).
